CMakeLists.txt

```diff
@@ -1294,10 +1294,17 @@ if(UNIX AND NOT APPLE)
       DESTINATION
         "${CMAKE_INSTALL_SYSCONFDIR}/udev/rules.d"
     )
-    message(WARNING "Installing udev rules to ${CMAKE_INSTALL_SYSCONFDIR}/udev/rules.d, "
-        "which is reserved for user-installed files. The correct directory for "
-        "system rules is /lib/udev/rules.d. with a appropiated priority prefix. "
-        "Adjust your package script accordingly and set -DINSTALL_USER_UDEV_RULES=OFF")
+    message(WARNING
+"The udev rule file for USB HID and Bulk controller permissions will be installed to "
```

Comment:
There are multiple formatting issues in this. Please reset this commit and merge my PR instead. Or just reset this commit and I'll open a followup PR to 2.3.

Reply:
Please rebase to remove this commit.